test/fixtures/served/iframe.html

```diff
@@ -52,6 +52,12 @@
             sentinel: sentinel,
             type: 'send-embed-state',
           }, '*');
+    } else if (event.data.type === 'requestPymjsHeight') {
+      var msg = [ 'pym', 'height', event.data.height ].join( 'xPYMx' );
+      getAmpWindow(sentinel)./*OK*/postMessage( msg, '*');
+    } else if (event.data.type === 'requestPymjsWidth') {
+      var msg = [ 'pym', 'width', event.data.height ].join( 'xPYMx' );
+      getAmpWindow(sentinel)./*OK*/postMessage( msg, '*');
```

Comment:
This `sentinel` I believe is `undefined` here, so that's not right. Nevertheless, I've tried populating `sentinel` here I tried populating with `event.data.sentinel` and `'amp'` and `'amp-test'` but no success.

Reply:
Hmm. I'll try to take a look tomorrow, but I'm running out of time a bit. Could you ping @jridgewell to see if he might have advise on messaging specifically for tests?

Reply:
The `sentinel` variable was indeed `undefined`. This seems to be part of the problem; the `var` was hoisting so the undefinedness wasn't causing an error. Fixed in c640f88.